packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -379,6 +380,64 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       })
     })
 
+    it('handles publishing a ProviderRebalancingZone policy intent', async () => {
+      const draft: ProviderRebalancingZoneIntentDraft = {
+        intent_type: 'provider_rebalancing_zone',
+        rule_fields: [
+          {
```

Comment:
One potential edge case here is when there are multiple allowed rules, so might be worth adding one more rule or another test case with multiple rules, but not sure that sort of validation needs to be done at the MDS core layer (might be covered by lacuna-core so is unnecessary here).